src/main/kotlin/com/cognifide/gradle/aem/config/Config.kt

```diff
@@ -45,7 +45,7 @@ class Config(
      * Default path is a subfolder named '.aem' under root project directory.
      */
     @Input
-    var instanceRoot: String = "${aem.project.rootProject.file(".aem")}"
+    var instanceRoot: String = aem.props.string("aem.config.instanceRoot") ?: "${aem.project.rootProject.file(".aem")}"
```

Comment:
its was intentionally not overriddable by gradle.properties. what is a use case?

Reply:
@fbatk was struggling with it trying to override this default. That's why I wanted to make it simple for users to do that. I think it should be specific for user, not stored in build script for the whole project.

Reply:
Ok, trusting you